src/Pangraph.hs

```diff
@@ -28,17 +28,17 @@ import Data.Maybe            (mapMaybe, isJust)
 import Data.Map.Strict       (Map)
 import qualified Data.Map.Strict  as Map
 import qualified Data.ByteString  as BS
-{-| The 'Pangraph' type is abstract and intended to be the core intermediate type between abstract representations of other graph libaries and the results of parsers. -}
+{-| The 'Pangraph' type is abstract and the core intermediate type between abstract representations of graphs. -}
 data Pangraph = Pangraph
   { vertices' :: Map VertexID Vertex
   , edges' :: Map EdgeID Edge
   } deriving (Eq)
-{-| A 'Vertex' holds `Attributes` and must have a unique `VertexID` to be constructed -}
+{-| A Vertex holds `Attributes` and must have a unique `VertexID` to be constructed with `makeVertex`. -}
```

Comment:
I don't think Haddock supports backticks? Use single quotes to link to other definitions, e.g. `holds a list of 'Attribute' values`, `'VertexID'` and `'makeVertex'`.